uber/models.py

```diff
@@ -401,6 +401,11 @@ def apply(self, params, *, bools=(), checkgroups=(), restricted=True, ignore_csr
                             value = None
                         else:
                             value = float(value)
+                    elif isinstance(column.type, Numeric):
+                        if value == '':
+                            value = None
+                        elif value.endswith('.0'):
+                            value = int(float(value))
```

Comment:
If we've got a `Numeric` column and we're storing a value with a trailing zero after the decimal point, go ahead and convert it to an `int`.

_Looking in your direction, `Group.tables`_ ಠ_ಠ